src/Microsoft.Health.Dicom.Core/Features/Store/StoreErrorResult.cs

```diff
@@ -0,0 +1,19 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.Store;
+
+public sealed class StoreErrorResult
+{
+    public StoreErrorResult(string error, bool isRequiredTag)
+    {
+        Error = error;
+        IsRequiredTag = isRequiredTag;
+    }
+
+    public string Error { get; }
+
+    public bool IsRequiredTag { get; }
```

Comment:
We use `required` here and `core` in the validation result builder - are we planning to converge on one name?

Reply:
Changed to RequiredCoreTags. Let me know if this works